packages/wdio-jasmine-framework/src/index.ts

```diff
@@ -536,4 +537,13 @@ declare global {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         interface Matchers<R, T> extends jasmine.Matchers<R> {}
     }
+
+    // Firstly variable '_wdioDynamicJasmineResult' gets value in packages/wdio-jasmine-framework/src/index.ts and then used here in wdio-utils as workaround for Jasmine
+    // eslint-disable-next-line no-var
+    var _wdioDynamicJasmineResult: WdioDynamicJasmineResult
+}
+
+interface WdioDynamicJasmineResult {
+    passedExpectations: [any, ...any[]]
+    failedExpectations: [any, ...any[]]
 }
```

Comment:
This should be sufficient
```suggestion
    var _wdioDynamicJasmineResult: any
}
```